packages/web/app/src/lib/kit/json.ts

```
@@ -0,0 +1,18 @@
+export namespace Json {
+  export const Primitive = z.union([z.string(), z.number(), z.boolean(), z.null()]);
+  export type Primitive = z.infer<typeof Primitive>;
+  export const isPrimitive = ZodHelpers.createTypeGuard(Primitive);
+
+  export const Value: z.ZodType<Value> = z.lazy(() =>
+    z.union([Primitive, z.array(Value), z.record(Value)]),
+  );
+  export type Value = Primitive | { [key: string]: Value } | Value[];
+  export const isValue = ZodHelpers.createTypeGuard(Value);
+
+  export const Object: z.ZodType<Object> = z.record(Value);
```

Biome (1.9.4)

[error] Do not shadow the global "Object" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)